### PR TITLE
docs/README

### PR DESCRIPTION
**Pull request by tenthwalker**
# Description
This pull request adds wins/challenges, media previews, and an abstract description, as well as standardizes spacing and implements requested reviewer updates..

# Checklist
- [x] My PR has an appropriately descriptive and concise title.
- [x] I ran the code locally and verified that there are no visible errors.
- [x] docs: My PR clearly describes which doc(s) I'm updating and why.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -60,13 +60,18 @@ Kevin Zolman
 
 
 # 💭 Abstract
-<!-- TBD -->
+This application is a Dungeons and Dragons Encounter Builder, with the intended user being a Dungeon Master. It allows the user to fill out a form to add encounter details including encounter name, party size, party level, summary, description, treasure & rewards, and monsters. Monsters from the D&D 5e compendium can be filtered and added based on size, name, hit points, and armor class attributes.
```
Comment (ericahagle) [inline docs/README.md:64]: Might want to also add a line about the user being able to preview encounters that they've built, and then click to see full details of each encounter.

File: docs/README.md [hunk 2]
```diff
@@ -113,10 +118,14 @@ After completing installation of the app...
 
 # Challenges & Wins
 ## 🚧 Challenges
-
+- New technology combined with new vocabulary and concepts
```
Comment (ericahagle) [inline docs/README.md:121]: Might want to add specifics (GraphQL, first full-stack team project)

File: docs/README.md [hunk 3]
```diff
@@ -113,10 +118,14 @@ After completing installation of the app...
...
-
+- New technology combined with new vocabulary and concepts 
+- Work-life balance
+- Minimizing scope creep 
 
 ## 🌟 Wins
-
+- Learning how to talk to a back end team to reach a common goal
+- Working with content that interests us
+- Further practice with consulting documentation
```
Comment (ericahagle) [inline docs/README.md:129]: Could add something about:
- excellent team communication and collaboration
- actually getting graphql to work properly in the app and stubbed in tests

**Review by ericahagle — CHANGES_REQUESTED** [review]
In addition to the little bits tied to specific spots in the code:
- Lines 20-21 include Mocha and Chai badges, but we aren't actually using that type of testing, so those can be removed.
- Also, I don't think we have a link to our Figma anywhere, just the badge saying that we used Figma, so that should probably be added, too. (https://www.figma.com/file/onraXBNfP8PeeiThF8Bla0/Tavern-Keeper-FE?type=design&node-id=13-2&mode=design&t=JJGPVCZ9i5aa8wde-0)

File: docs/README.md [hunk 4]
```diff
@@ -60,13 +58,27 @@ Kevin Zolman
...
 
 # 📝 Context
-This application was built as the group capstone project of Turing School of Software and Design's Front End Web Development program, Mod 4. We were given 2 weeks, from kick-off, to complete and submit the project for evaluation. This repo houses the FE contribution to the project; the BE repo can be found here: [Tavern Keeper BE Repo][backend-repo-url]. The deployed API can be found here: [Tavern Keeper BE Deployment][backend-deployment-url].
+This application was built as the group capstone project of Turing School of Software and Design's Front End Web Development program, Mod 4. We were given 2 weeks from kick-off to complete and submit the project for evaluation. This repo houses the FE contribution to the project; the BE repo and installation instructions can be found here: [Tavern Keeper BE Repo][backend-repo-url]. The deployed API can be found here: [Tavern Keeper BE Deployment][backend-deployment-url].
+
 
 # 🎬 Preview
-<!-- TBD -->
+<details>
+<h2  id="preview"> 🎥 Desktop Preview 📱 </h2>
```
Comment (ericahagle) [inline docs/README.md:70]: Probably don't need the iphone emoji on the Desktop Preview h2.

File: docs/README.md [hunk 5]
```diff
@@ -60,13 +58,27 @@ Kevin Zolman
...
+<h3>MacBook</h3>
+![macbooktk](https://github.com/Tavern-Keeper-2308/tavern-keeper-fe/assets/139941423/83ab5706-193f-4ee9-a16a-ed9b0c105de5)
+
+<h2  id="preview"> 📱 Mobile and Tablet Views </h2>
+<h3>iPad</h3>
+![ipadtk](https://github.com/Tavern-Keeper-2308/tavern-keeper-fe/assets/139941423/7311f5eb-fdae-408e-b75c-66390a7dae8c)
+
+<h3>iPhone</h3>
+![iphonetk](https://github.com/Tavern-Keeper-2308/tavern-keeper-fe/assets/139941423/5b0aacce-ac4b-4b3e-81bf-0fcc05b0aad6)
+</details>
```
Comment (ericahagle) [inline docs/README.md:80]: None of these previews are actually rendering for me. Are they showing up on the README preview on your machine? All I see are the links themselves.

**Review by ericahagle — CHANGES_REQUESTED** [review]
Everything else looks good, aside from the two small bits I mentioned.

File: docs/README.md (hunk 5 above)
Comment (tenthwalker) [reply docs/README.md:80]: Nope, looking at it again they are being uncooperative. Posting them here in the comment to get the links to try again.
macbooktk
iphonetk
ipadtk

**Review by ericahagle — COMMENTED** [review]
@tenthwalker Hopefully the change I just pushed up has them working. It seems to be working on the branch on my machine. If so, you can remove the extra files that you added to the project, since we won't need them. Then we can merge this in. 

It was a very silly fix, btw. I had no idea and just stumbled upon the solution while I was messing around with it. For whatever reason, there needed to be linespace around the lines that have the links on them. I am baffled by .md files. 😂 😭

**Review by ericahagle — APPROVED** [review]
Looks great! 🍪

**Pull request merged**